README.md

```diff
@@ -36,17 +36,19 @@ Core Server (i.e. SERVER) project are **public**.
 ## Installation
 `MongoSwift` works with Swift 4.0+.
 
-### OS X and Linux
+### macOS and Linux
 
-Installation on OS X and Linux is supported via [Swift Package Manager](https://swift.org/package-manager/).
+Installation on macOS and Linux is supported via [Swift Package Manager](https://swift.org/package-manager/).
 
 #### Step 1: Install the MongoDB C Driver
 The driver wraps the MongoDB C driver, and using it requires having the C driver's two components, `libbson` and `libmongoc`, installed on your system. **The minimum required version of the C Driver is 1.13.0**.
 
-On a Mac, you can install both components at once using [Homebrew](https://brew.sh/):
+*On a Mac*, you can install both components at once using [Homebrew](https://brew.sh/):
 `brew install mongo-c-driver`.
 
-On Linux: please follow the [instructions](http://mongoc.org/libmongoc/current/installing.html#building-on-unix) from `libmongoc`'s documentation. Note that the versions provided by your package manager may be too old, in which case you can follow the instructions for building and installing from source.
+*On Linux*: please follow the [instructions](http://mongoc.org/libmongoc/current/installing.html#building-on-unix) from `libmongoc`'s documentation. Note that the versions provided by your package manager is probably too old, in which case you can follow the instructions for building and installing from source.
```

Comment:
nitpick: "the versions provdided by your package manager _are_ probably too old..."